operations/mimir/config.libsonnet

```diff
@@ -69,6 +69,7 @@
     storage_azure_account_key: error 'must specify Azure account key',
 
     jaeger_agent_host: null,
+    querier_jaeger_reporter_max_queue_size: 5000,
```

Comment:
Don't think we need to allow to configure it. An user can always override it in `querier_env_map`.